sdk/tests/conformance2/wasm/bufferdata-2gb-in-4gb-wasm-memory.html

```diff
@@ -0,0 +1,64 @@
+<!--
+Copyright (c) 2024 The Khronos Group Inc.
+Use of this source code is governed by an MIT-style license that can be
+found in the LICENSE.txt file.
+-->
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>bufferData test to Wasm Memory 4GB in size.</title>
+<link rel="stylesheet" href="../../resources/js-test-style.css"/>
+<script src="../../js/js-test-pre.js"></script>
+<script src="../../js/webgl-test-utils.js"> </script>
+</head>
+<body>
+<canvas id="canvas" width="2" height="2" style="width: 40px; height: 40px;"></canvas>
+<div id="description"></div>
+<div id="console"></div>
+<script>
+"use strict";
+description(document.title);
+debug("Tests that bufferData can be called on a ~2GB view into ~4GB of WebAssembly Memory.");
```

Comment:
This sounds ambiguous to me, like the view itself is 2 GB in length - could you consider a rephrase? Something like "called on a small view which is ~2 GB into a ~4 GB WebAssembly Memory"?